java/pom.xml

```diff
@@ -69,43 +69,37 @@
       <dependency>
         <groupId>com.google.http-client</groupId>
         <artifactId>google-http-client</artifactId>
-        <version>1.28.0</version>
-        <exclusions>
-          <exclusion>
-            <artifactId>guava-jdk5</artifactId>
-            <groupId>com.google.guava</groupId>
-          </exclusion>
-        </exclusions>
+        <version>1.32.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.google.http-client</groupId>
         <artifactId>google-http-client-protobuf</artifactId>
-        <version>1.28.0</version>
+        <version>1.32.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.google.http-client</groupId>
-        <artifactId>google-http-client-jackson</artifactId>
-        <version>1.28.0</version>
+        <artifactId>google-http-client-jackson2</artifactId>
+        <version>1.32.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.google.oauth-client</groupId>
         <artifactId>google-oauth-client</artifactId>
-        <version>1.28.0</version>
+        <version>1.30.4</version>
       </dependency>
 
       <dependency>
         <groupId>com.google.api-client</groupId>
         <artifactId>google-api-client</artifactId>
-        <version>1.28.0</version>
+        <version>1.30.5</version>
```

Comment:
Make these numbers (such as 1.30.5) variable(s)

Reply:
The only ones that are duped are now pulled in via a BOM.